src/main/java/examples/Examples.java

```diff
@@ -2,10 +2,10 @@
 
 // Import classes:
 import io.tiledb.cloud.*;
-import io.tiledb.cloud.rest_api.ApiException;
-import io.tiledb.cloud.rest_api.api.GroupsApi;
-import io.tiledb.cloud.rest_api.api.ArrayApi;
-import io.tiledb.cloud.rest_api.model.*;
+import io.tiledb.cloud.rest_api.v1.ApiException;
+import io.tiledb.cloud.rest_api.v1.api.GroupsApi;
+import io.tiledb.cloud.rest_api.v1.api.ArrayApi;
+import io.tiledb.cloud.rest_api.v1.model.*;
```

Comment:
This will be a breaking change. Not sure if you want that.

Reply:
It will be a breaking change yes, necessary in my opinion to include the new code. Otherwise the packaging will be unclear

Reply:
I see. I'm leaving it to others to make the decision on the breaking change.

Reply:
Per discussion, we're going to alias the v1 elements and provide a deprecation pathway.

Reply:
Class aliases are not available in Java but I did the following. I created the v1 directory and also left the v1 files in their previous directory. So I pretty match did a copy-paste and we ended up with multiple duplicates. In the files that are in the unwanted directory I added a deprecation warning. These files will be removed in our next release.

Reply:
The deprication warning is:

```
**
  * @deprecated This class is now moved to io.tiledb.cloud.rest_api.v1.api
  */
 @Deprecated
```

Reply:
Thanks. Without an aliasing mechanism (java 🙃), this will be an all-or-nothing change: users will need to update all uses at once, because the `v1.*` types will be semantically distinct even if they are structurally identical. So, if at all possible, we should wait two libTileDB releases before removing the old signatures in order to follow our general deprecation policy.

Reply:
sounds good!